.github/workflows/4-standalone.yaml

```diff
@@ -65,6 +65,8 @@ jobs:
           sudo rm -rf "$AGENT_TOOLSDIRECTORY"
       - name: Check free space
         run: df -h
+      - name: Disable the scheduler
+        run: "./tools/github/disable-scheduler.sh"
```

Comment:
The standalone mode is only supported with ShardingPoolBalancer, so we need to disable the scheduler for standalone tests.

Reply:
No objections to the change, but do you think the standalone mode could be supported with the new scheduler or should we be looking to deprecate standalone mode?  Long term, I'd think we would want to consolidate down to a single scheduler (the new one) to simplify maintenance.

Reply:
Yes, that would be great to discuss and we can handle it in another issue if necessary.
I just wanted not to postpone the release of the new version of OpenWhisk Core due to this.

I am still unclear if we can support the standalone mode with the scheduler or even if it would be efficient to enable the scheduler in the standalone mode.
The scheduler works as a queue for actions and serves activations according to requests from containers(invokers).
Currently, a controller works as a controller and an invoker at the same time in the standalone mode.
If we put the scheduler into it, it would balance loads, provision containers, buffer activations, and send them to containers. We can make it work but I am not sure making one component that is in charge of all of these roles is aligned well with the goal of the standalone mode. I expect the standalone mode would be used in an environment with limited resources like an IoT machine. I have a concern that it might be too complex and require too many resources in such a circumstance.

Reply:
Agree @dgrove-oss +1